src/components/app.scss

```diff
@@ -21,5 +21,6 @@
   height: fit-content;
   padding: 0 20px;
   background-image: linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0));
+  line-height: 1.35;
+  color: #444;
```

Comment:
should this be a predefined color in vars?

Reply:
Added both to the vars file. #444 as medium-font-color. line height as line-height.